test/model.user-identity.test.js

```diff
@@ -239,4 +239,60 @@ describe('UserIdentity', function() {
       done();
     });
   });
+
+  describe('emailOptional set to false', function(err) {
+    it('creates a user when given an email address', function(done) {
+      var username = 'fbu123';
+      UserIdentity.login('facebook', 'oAuth 2.0', {
+        emails: [{value: 'abc123@example.com'}],
```

Comment:
I mean can you put `emails: {value: 'abc123@example.com'}`, meaning can we simplify it to take ONE object directly instead of an Array with one object.